Sources/GRPCCore/GRPCClient.swift

```diff
@@ -208,7 +208,7 @@ public final class GRPCClient: Sendable {
   ///
   /// The client, and by extension this function, can only be run once. If the client is already
   /// running or has already been closed then a ``RuntimeError`` is thrown.
-  public func run() async throws {
+  public func maintainConnections() async throws {
```

Comment:
Minor:
I (think I) understand why `maintainConnection` instead of `connect` - because it's async thus a long-running task and the act of just connecting doesn't imply it runs for a long period of time. But I find the naming still a bit weird. Did you consider something like `connectAndRun()`?

Reply:
That's fair, I wasn't 100% happy with the name.

Broadly speaking this function is:
- long running
- meant to be run in the background
- will manage any underlying connections

I think `maintainConnections` does an _okay_ job at capturing that. Some alternatives:
- `connectAndRun`
- `runBackgroundWork`
- `runBackgroundTasks`
- `manageConnections`
- `runConnections`

WDYT?

Reply:
I think `runConnections`, `manageConnections` and (I'll add a third one) `startAndRunConnections` achieve the three goals you listed the best.